test/helpers.ts

```diff
@@ -22,58 +20,27 @@ export function depSort(a, b) {
   return 0;
 }
 
-export async function graphToDepTree(depGraphInterface: depGraphLib.DepGraph): Promise<DepTree> {
-
-  const depGraph = (depGraphInterface as types.DepGraphInternal);
-
-  // TODO: implement cycles support
-  if (depGraph.hasCycles()) {
-    throw new Error('Conversion to DepTree does not support cyclic graphs yet');
+export function depTreesEqual(a, b) {
+  if (a.name !== b.name || a.version !== b.version) {
+    return false;
   }
 
-  const depTree = await buildSubtree(depGraph, depGraph.rootNodeId);
+  const aDeps = a.dependencies || {};
+  const bDeps = b.dependencies || {};
 
-  (depTree as any).packageFormatVersion = constructPackageFormatVersion(depGraph);
-
-  return depTree;
-}
-
-function constructPackageFormatVersion(depGraph: types.DepGraph): string {
-  let packageManagerShorthand = depGraph.pkgManager.name;
-  if (depGraph.pkgManager.name === 'maven') {
-    packageManagerShorthand = 'mvn';
+  if (_.keys(aDeps).sort().join(',') !== _.keys(bDeps).sort().join(',')) {
```

Comment:
`Object.keys()` over lodash?

Reply:
need to use lodash because targeting es5